pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -117,6 +118,14 @@ def __getstate__(self):
         state["predict_progress_bar"] = None
         return state
 
+    @staticmethod
+    def _resolve_refresh_rate(refresh_rate: int) -> int:
```

Comment:
Should this method be part of the `ProgressBarBase` directly ?

Reply:
Nope, as it works well with RichProgressBar

Reply:
can we keep the order properties, public methods, protected methods?